src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -63,9 +65,7 @@ public static LockFileTargetLibrary CreateLockFileTargetLibrary(IPackage package
             // it has the correct casing that runtime needs during dependency resolution.
             lockFileLib.Name = correctedPackageName ?? package.Id;
             lockFileLib.Version = package.Version;
-
-            var files = package.GetFiles().Select(p => p.Path.Replace(Path.DirectorySeparatorChar, '/')).ToList();
-
+            var files = library.Files.Select(p => p.Replace(Path.DirectorySeparatorChar, '/'));
```

Comment:
Path.Combine does alot of work. I'll do this in another PR, when the Files node gets read from the lock file, it does some path resolution and that might have a ripple effect on other parts of the code. This was tested and works.

I'll take a look at the other part later.